libs/dumb-components/src/lib/Slider.tsx

```diff
@@ -46,20 +57,32 @@ const StyledSlider = styled(ReactSlider)`
 `
 
 const renderThumb: FC = props => <StyledThumb {...props} />
-const renderTrack: FC = (props, state) => <StyledTrack {...props} index={state.index} />
+const renderTrack: FC = (props, state) => <StyledTrack {...props} index={state.index} value={state.value} />
 const renderMark: FC = props => <StyledMark {...props} />
 
-export const Slider: FC<Props> = ({ min, max, value, step, intervals = 5, onChange }) => {
+export const Slider: FC<Props> = ({ className, min, max, value = 0, step, intervals = 5, onChange, disabled = false }) => {
   const rangeBound = max - min
   const interval = rangeBound / intervals
   const markRange = intervals ? Array.from(Array(intervals - 1).keys()).map(i => min + interval * (i + 1)) : undefined
+  const ref = useRef<any>(null)
+
+  // Workaround for https://github.com/zillow/react-slider/issues/214
+  useEffect(() => {
+    if (!ref?.current) return
+    ref?.current?.handleResize?.()
+    return () => {}
+  }, [ref, onChange])
+
   return (
     <StyledSlider
+      ref={ref}
+      disabled={disabled}
+      className={className}
       marks={markRange}
       renderTrack={renderTrack}
-      renderThumb={renderThumb}
+      renderThumb={(!disabled && renderThumb) || undefined}
       renderMark={renderMark}
-      onChange={v => onChange(v as number)}
+      onChange={v => onChange?.(v as number)}
```

Comment:
this looks odd, no? what's the case where `onChange` won't exist?

Reply:
I also don't see that case, I'll make it required